src/main/kotlin/org/rust/ide/annotator/RsHighlightingAnnotator.kt

```diff
@@ -42,26 +42,35 @@ class RsHighlightingAnnotator : AnnotatorBase() {
         holder.newSilentAnnotation(severity).textAttributes(color.textAttributesKey).create()
     }
 
+    // For elements in a RsMacroExpansionReferenceGroup or RsMacroBindingGroup, return a macro color.
+    // Otherwise, return null
+    // Only the PARENT of the element is passed.
```

Comment:
nit: Not sure that this addition is helpful

Reply:
Removed.